docs/src/contribute/BUILDING.md

```diff
@@ -12,7 +12,7 @@ You can use any OS to build from source (below has been tested on Ubuntu, but we
 
 Our recommended setup and required tools are:
 
-* Ubuntu 20.04 or Debian 10.9 (Buster)
+* Ubuntu 20.04 or Debian 12 (Bookworm)
```

Comment:
We may have to increase that minimum Ubuntu version as well, as its inbuilt repositories have an older version of mosquitto: `1.9.1` IIRC, unless we expect even the users to configure the sid repo and update to the exact version that we recommend.

Reply:
though I guess that is only regarding building the project and not specifically using it

Reply:
But yes I think we need to update all of the build instructions to also use the newer way of doing things using zigbuild and the just(file) recipes.

Reply:
Aah okay, just noticed that it's `BUILDING.md` and not `supported-platforms.md`, where we document the officially supported versions, which still refers to Ubuntu 20.04.